packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -87,4 +94,34 @@ export class TableInputCSV extends React.PureComponent<Props, State> {
   }
 }
 
-export default TableInputCSV;
+export const TableInputCSV = withTheme(UnThemedTableInputCSV);
+TableInputCSV.displayName = 'TableInputCSV';
+
+interface TableInputCSVStyle {
+  tableInputCsv: string;
+  textarea: string;
+  footer: string;
+}
+
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme): TableInputCSVStyle => {
```

Comment:
FYI- TS did a good job so far inferring the return type of the style getters, so I think you should not need return type annotation:)

Reply:
Alright. would update right away